libs/couch2pg.js

```diff
@@ -10,20 +10,21 @@ const parseSource = url => {
   return `${source.host}${source.path}`;
 };
 
-const replicate = (couchUrl, pgconn, opts) => {
+const replicate = (couchUrl, pgconn, opts, table) => {
   return couch2pg.importer(
           pgconn,
           new PouchDB(couchUrl),
           opts.docLimit,
           opts.changesLimit,
-          parseSource(couchUrl)).importAll();
+          parseSource(couchUrl),
+          table).importAll();
 };
 
 const migrate = (pgUrl) => {
   return couch2pg.migrator(pgUrl)();
 };
 
 module.exports = {
-  replicate: (couchUrl, pgconn, opts) => replicate(couchUrl, pgconn, opts),
+  replicate: (couchUrl, pgconn, opts, table = 'couchdb') => replicate(couchUrl, pgconn, opts, table),
```

Comment:
I think we should remove the default option here and hardcode 'couchdb' into the calling functions so it's explicit what table is being used.